lib/wallet.js

```diff
@@ -122,8 +122,11 @@ function mergeStatusMode (a, b) {
 }
 
 function getWalletStatus (settings, tx) {
+  const fudgeFactorEnabled = configManager.unscoped(settings.config).fudgeFactor
+  const fudgeFactor = fudgeFactorEnabled ? 10 : 0
+
   const walletStatusPromise = fetchWallet(settings, tx.cryptoCode)
-    .then(r => r.wallet.getStatus(r.account, tx.toAddress, tx.cryptoAtoms, tx.cryptoCode))
+    .then(r => r.wallet.getStatus(r.account, tx.toAddress, tx.cryptoAtoms.add(fudgeFactor), tx.cryptoCode))
```

Comment:
Don't we want to subtract the fudgeFactor here?

Reply:
Good catch, this was previously inside each wallet, hence the error.